xija/component/heat.py

```diff
@@ -672,24 +672,21 @@ class AcisDpaStatePower(PrecomputedHeatPower):
     state.  See dpa/NOTES.power.
     """
     def __init__(self, model, node, mult=1.0,
-                 fep_count=None, ccd_count=None,
-                 vid_board=None, clocking=None):
+                 pow_states=None, fep_count=None, 
```

Comment:
It would probably be best to put the new kwarg at the end just in case this class is being initialized with positional arguments somewhere.  Probably unlikely, but you never know.